gst/nnstreamer/tensor_query/tensor_query_server.c

```diff
@@ -78,7 +79,9 @@ gst_tensor_query_server_add_data (char *id, const gchar * topic)
   else
     _topic = g_strdup ("TCP_DIRECT");
 
-  ret = nns_edge_create_handle (id, _topic, &data->edge_h);
+  ret =
+      nns_edge_create_handle (id, connect_type, NNS_EDGE_FLAG_ALL,
```

Comment:
same here, plz do not use NNS_EDGE_FLAG_ALL for future work.
(NNS_EDGE_FLAG_RECV | NNS_EDGE_FLAG_SEND | NNS_EDGE_FLAG_SERVER)